srv/salt/_modules/rgw.py

```diff
@@ -125,6 +125,16 @@ def create_bucket(**kwargs):
         return False
     return True
 
+def lookup_bucket(user, bucket):
+    s3conn = boto.connect_s3(
```

Comment:
I would clean up as suggested by @theanalyst and make one single connection class and pass all the required parameters. Will update PR accordingly.